SPECIFICATION.md

```diff
@@ -101,17 +101,23 @@ the number may be omitted. If the integer part is zero and there is a fractional
 part, the zero may be omitted (e.g. `-.5` instead of `-0.5`).
 
 The coordinate system starts at the lower left corner. Positive X coordinates
-point to the right. Positive Y coordinates point to the top. Negative
-coordinates are not allowed.
+point to the right. Positive Y coordinates point to the top. A value of 9 means
+100% of the font height (i.e. the coordinate system is scaled with factor 9).
+This system leads to more compact font files as many coordinates can be
+expressed by a single digit.
 
           Y
-    (0,Y) +
+    (0,9) +
           |
           |
           +-----+ X
-      (0,0)   (X,0)
+      (0,0)   (9,0)
+
+Typical glyphs use the X axis (Y=0) as the baseline and are drawn inside the
+area (0,0)..(9,9). Coordinates outside this area are allowed, e.g. for ascenders
+and descenders. And typical glyphs are left aligned to the Y axis (X=0).
+Otherwise it influences letter spacing, see section "Letter Spacing" for details.
```

Comment:
Maybe write "Glyphs that are not aligned to the Y axis have an impact on letter spacing, (...)" instead?